### PR TITLE
[FLINK-9109] [doc] Update documentation for CLI

### PR DESCRIPTION
**Pull request by tillrohrmann**
## What is the purpose of the change

Update documentation for CLI.

### CODE REVIEW

File: docs/ops/cli.md [hunk 1]
```diff
@@ -300,70 +306,134 @@ Action "info" shows the optimized execution plan of the program (JSON).
...
   Options for yarn-cluster mode:
+     -m,--jobmanager <arg>            Address of the JobManager (master) to
+                                      which to connect. Use this flag to connect
+                                      to a different JobManager than the one
+                                      specified in the configuration.
      -yid,--yarnapplicationId <arg>   Attach to running YARN session
+     -z,--zookeeperNamespace <arg>    Namespace to create the Zookeeper
+                                      sub-paths for high availability mode
+
+  Options for flip6 mode:
```
[the remark this answers is not in the document]
Comment (tillrohrmann) [reply docs/ops/cli.md]: Good point @zentol. Have to update the PR accordingly.

**Comment by zorro786** [comment]
@tillrohrmann Why is passing job-manager option in standalone mode not supported now?

**Comment by zentol** [comment]
it is listed separately under the flip6 options in the version of this PR.

**Comment by zorro786** [comment]
@zentol What if I want to pass jobmanager option in non flip6 mode?

**Comment by zentol** [comment]
@zorro786 That should still work. But you're right that it should also be listed, maybe as a "legacy" option or the separate flip6 option should be removed.

**Comment by tillrohrmann** [comment]
The section `flip6` is effectively the standalone mode @zorro786. I'll update the documentation accordingly. The reason why the job manager option is now shown under the different `CustomCommandLines` is that it effectively is an option relevant for the interaction with a remote cluster and, thus, an implementation of the `CustomCommandLine`. Theoretically, different Flink clusters, e.g. Yarn and Mesos, might implement different means to identify a Flink cluster. For example, the Yarn application id or an URL.

**Comment by tillrohrmann** [comment]
I've updated the PR @zorro786 and @zentol.

File: docs/ops/cli.md [hunk 2]
```diff
@@ -287,6 +287,16 @@ Action "run" compiles and runs a program.
                                           sub-paths for high availability mode
      -ynl,--yarnnodeLabel <arg>           Specify YARN node label for 
                                           the YARN application 
+     -z,--zookeeperNamespace <arg>        Namespace to create the Zookeeper
+                                          sub-paths for high availability mode
+
+  Options for default mode:
```
Comment (zentol) [inline docs/ops/cli.md]: Can we list this above the yarn options?

**Pull request closed without merging**